src/components/dailyQuestsPage/SuggestedQuests.jsx

```diff
@@ -0,0 +1,174 @@
+    } else {
+      async function fetchSuggestions() {
+        try {
+          const response = await fetch("/api/v1/daily-quests/suggestions", {
+            headers: {
+              Authorization: `Bearer ${yourTokenHere}`,
+            },
+          });
+          const data = await response.json();
+          setSuggestions(data);
+        } catch (error) {
+          console.error("Error fetching suggestions:", error);
+        }
+      }
+
+      fetchSuggestions();
+    }
+  }, []);
+
+  const filteredSuggestions = suggestions.filter((suggestion) => {
+    return (
+      !quests.some((quest) => quest.title === suggestion.title) &&
+      !userQuests.some((quest) => quest.title === suggestion.title)
+    );
+  });
+
+  function handleRefresh() {
+    if (isLocal) {
+      const availableSuggestions = initialSuggestions.filter((suggestion) => {
+        return !quests.some((quest) => quest.title === suggestion.title);
+      });
+
+      const shuffled = [...availableSuggestions].sort(
+        () => 0.5 - Math.random(),
+      );
+      setSuggestions(shuffled.slice(0, Math.min(3, shuffled.length)));
+    } else {
+      async function fetchSuggestions() {
+        try {
+          const response = await fetch("/api/v1/daily-quests/suggestions", {
+            headers: {
+              Authorization: `Bearer ${yourTokenHere}`,
+            },
+          });
+          const data = await response.json();
+          setSuggestions(data);
+        } catch (error) {
+          console.error("Error fetching suggestions:", error);
+        }
+      }
+
+      fetchSuggestions();
+    }
+  }
+
+  return (
+    <div className="py-4 w-full mx-auto bg-transparent">
+      <div className="flex items-center mb-4">
+        <h2 className="text-sm font-semibold text-gray-500 flex-grow">
+          Suggestions
+        </h2>
+        <div className="flex gap-1">
+          <button
+            onClick={() => setShowSuggestions((prev) => !prev)}
+            className="w-5 h-5 flex items-center justify-center text-gray-500 hover:opacity-70 transition"
+            aria-label="Toggle Suggestions"
+            type="button"
+          >
+            {showSuggestions ? (
+              <EyeIcon className="w-4 h-4" />
+            ) : (
+              <EyeSlashIcon className="w-4 h-4" />
+            )}
+          </button>
+          <button
+            onClick={handleRefresh}
+            className="w-5 h-5 flex items-center justify-center text-gray-500 hover:opacity-70 transition"
+            aria-label="Refresh Suggestions"
+            type="button"
+          >
+            <ArrowPathIcon className="w-4 h-4" />
+          </button>
+        </div>
+      </div>
+      {showSuggestions && (
+        <div>
+          {filteredSuggestions.length > 0 ? (
+            <ul className="space-y-1.5 w-full">
+              {filteredSuggestions.map((s, index) => (
+                <li
+                  key={index}
```

Comment:
it's a bad practice and should be done if there's no other options. You have options. Suggestion has fields that you can use